toolbox/linux/nowwhat.rst

```diff
@@ -0,0 +1,127 @@
+I Installed It!  Now What?
+==========================
+
+Congratulations! If everything worked properly, you should be looking at a
+freshly installed Linux login screen, either inside a virtual machine or "for
+real for real" on your main computer.
+
+What to do from here is up to you. The Toolbox pages after this one give some
+great things to learn to do on Linux. However, here are some other things you
+may want to do besides the technical stuff.
+
+*Note that the commands in code blocks below should be run in your Terminal
+program on Linux.  Also, these assume you installed Ubuntu.*
+
+How Do I Install Stuff?
+-----------------------
+
+Every Linux distribution has its own package manager, and so the instructions
+may be slightly different depending on your distro. If you did Ubuntu, all you
+need to do is run:
+
+.. code::
+
+   sudo apt install <package name>
+
+Typically the package name is exactly what you'd expect, but lower case. For
+example, ``git``, ``python``, ``firefox``, etc. However, sometimes it may have a
+slighly different name from expected. You can search for packages `online
+<http://packages.ubuntu.com>`_.
```

Comment:
You can also use `apt-get` to search for packages, and just be lazy and use tab completion. Not sure if we want to add that last one as an actual suggestion though :)